kratos/version/kratos_version.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
```

Comment:
I thought `#pragma once` was not working properly for some configurations thats why we use the IFDEF stuff?

Reply:
Indeed, `#pragma once` is compiler dependent

Reply:
Well its true but I was under the impression that 99% of compilers support it. At least I can grantee you that gcc(3.4 or greater), clang, intel and MSVC do support it.

Technically, any pragma is (or should be) part of the standard, as they solo propose is be something out of the standard.

As a matter of fact, there are several files that are already using this (all pybind, amatrx, parts of the meshing...)

The real problem with this particular pragma is that if one has the same version of the file in two different locations (for example via links or because one fs is mounted over different directories) the compiles will be unable to differentiate those files.

Still I can change if its clearer with the defines.

Reply:
I start to use #pragma once as it is very clear and don't have the typical problem of if defs with different parameters due to wrong copy paste or files with the same macro due to cpp save as.

What is the problem with it?